### PR TITLE
API provides action mailer functionality 

### PR DESCRIPTION
**Pull request by oliverochman**
PT https://www.pivotaltracker.com/story/show/178351132

### User story
```
As an API
In order to notify Flex Coast about new inquiries
I want to send new inquiries off to their email
```

### Changes proposed
- Send email to broker when visitor submits new inquiry

### What i have learned working on this feature
- How Action mailer works
- Usage of after hook

### CODE REVIEW

File: app/views/inquiry_mailer/created_email.html.erb [hunk 1]
```diff
@@ -0,0 +1,31 @@
+<html>
```
Comment (DavDan1) [inline app/views/inquiry_mailer/created_email.html.erb:1]: is this a templete?

File: config/routes.rb [hunk 2]
```diff
@@ -1,5 +1,5 @@
 Rails.application.routes.draw do
   namespace :api do
-    resources :inquiries, only: [:create, :index, :update]
+    resources :inquiries, only: %i[create index update]
```
Comment (DavDan1) [inline config/routes.rb:3]: try to format before commiting

**Review by DavDan1 — APPROVED** [review]

File: app/views/inquiry_mailer/created_email.html.erb (hunk 1 above)
Comment (oliverochman) [reply app/views/inquiry_mailer/created_email.html.erb:1]: This is how the email will look, used a very simple layout. Can be styled in the future.

File: spec/requests/api/visitors_can_create_inquiries_spec.rb [hunk 3]
```diff
@@ -43,7 +92,7 @@
                peers: true,
                email: '',
                flexible: true,
-               phone: 0707123456,
+               phone: 0o707123456,
```
Comment (tochman) [inline spec/requests/api/visitors_can_create_inquiries_spec.rb]: 🤨

Comment (christerforsgren91) [reply spec/requests/api/visitors_can_create_inquiries_spec.rb]: formatter does this sh**t!

File: config/routes.rb (hunk 2 above)
Comment (christerforsgren91) [reply config/routes.rb:3]: Oliver dont have a formatter, so kim pulled it down and formatted it!

File: app/views/inquiry_mailer/created_email.html.erb (hunk 1 above)
Comment (DavDan1) [reply app/views/inquiry_mailer/created_email.html.erb:1]: sweet!

File: config/routes.rb (hunk 2 above)
Comment (DavDan1) [reply config/routes.rb:3]: okay. Thanks for your inquiry

**Review by DavDan1 — APPROVED** [review]

**Review by lukeperrin10 — APPROVED** [review]

**Review by arunbhalli — APPROVED** [review]

File: spec/requests/api/visitors_can_create_inquiries_spec.rb (hunk 3 above)
Comment (tochman) [reply spec/requests/api/visitors_can_create_inquiries_spec.rb]: Save a phone number as a string instead

**Pull request merged**